shared-src/docs-style-guide.rst

```diff
@@ -1036,7 +1036,7 @@ XLSForm
 
 - The `XLSForm format for describing form in an Excel spreadsheet <http://xlsform.org/>`_
 - A spreadsheet file that describes a form using the format.
-- An `online tool <https://docs.opendatakit.org/xlsform/>`_ and an `offline tool <https://gumroad.com/l/xlsform-offline>`_ for converting :file:`*.xls(x)` files to XForm documents. 
+- A `tool </xlsform/>`_ for converting :file:`*.xls(x)` files to XForm documents.
```

Comment:
Should use the doc directive here.

Reply:
i *think* cross reference links work for shared src stuff... (If i remember correctly, it ends up getting resolved via intersphinx)

Reply:
Hmm, @yanokwa, there was an error when you tried it, right? Do you still know what the error was?

Reply:
Finally figured it out. For future reference...

failed:
```
:doc:`tool </xlsform>`
```

worked: 
```
:doc:`tool <xlsform>`
```

Reply:
👍 Thanks @adammichaelwood for letting us know it was possible! Also, it looks much better without the `/` -- I didn't even recognize it as a link at first because of that.